### PR TITLE
Feature/heroicon vue

### LINKED ISSUE #242: [Features] Use of Heroicons vue to uniform the icone interfaces.
## Description

We Should use Heroicons Vue to avoid creating svg icone component.
And we will have an uniform interface.

## Acceptance Criteria

- [x] Installed HeroIcons Vue
- [x] Refactor icon to use HeroIcons

### PR DESCRIPTION
**Pull request by hermannleboss**
# Description

Use of Hero Icon to avoid the usage of custom SVG component
Fixes #242 

## Type of change

Refactoring

## Acceptance Criteria

- [x] Installed HeroIcons Vue
- [x] Refactor icon to use HeroIcons

### CODE REVIEW

**Review by dasarathg68 — COMMENTED** [review]
Add Member plus icon color seems to be wrong
image

File: app/src/components/forms/AddMemberForm.vue
Comment (dasarathg68) [inline app/src/components/forms/AddMemberForm.vue:1]: why is the class text-green? we dont have any text there
it should be bg-green-700 right?

**Review by dasarathg68 — APPROVED** [review]
LGTM

File: app/src/components/forms/AddMemberForm.vue
Comment (hermannleboss) [reply app/src/components/forms/AddMemberForm.vue:1]: TO change icon color you use colors in css what is text-[color]

Comment (hermannleboss) [reply app/src/components/forms/AddMemberForm.vue:1]: I will skip the colors for the icons in the PR. 

We will focus on them later.

**Pull request merged**